tests/tests_unit/test_data_classes/test_data_models/test_typed_instances.py

```diff
@@ -395,16 +426,12 @@ def test_cognite_asset_read_and_write__to_pandas(
             assert df.index[1] == xid
             assert df.at["type", "value"] == {"space": "should-be", xid: "at-root"}
 
-        assert not read_expanded_df.index.is_unique  # because 'type' is repeated
-        expected_type_df = pd.DataFrame(
-            [
-                ({"space": "should-be", xid: "at-root"},),
-                ({"space": "should-be", "externalId": "in-properties"},),
-            ],
-            columns=["value"],
-            index=["type", "type"],
+        expected_type_series = pd.Series(
+            [{"space": "should-be", xid: "at-root"}],
+            index=["value"],
+            name="type",
         )
-        pd.testing.assert_frame_equal(read_expanded_df.loc["type"], expected_type_df)
+        pd.testing.assert_series_equal(read_expanded_df.loc["type"], expected_type_series)
```

Comment:
After these changes, this is now already covered on line 427. Perhaps we add a check on `asset_type` (or what the alias name was) instead?

Reply:
Fixed.